book/platform/examples/correct/hello-world/lib/hello.mli

```diff
@@ -0,0 +1,27 @@
+(* This file is the interface (aka exposed API, or signature) for the [Hello]
+   module. It only serves as documenting the module and selecting the functions
+   to expose. The actual implementation of the functions are located in the
+   module implementation [hello.ml]. *)
+
+(** This is a docstring, as it starts with "(**", as opposed to normal comments
```

Comment:
This comment should probably be changed as it contains the synopsis of the module and will be spliced into the namespace module page.

Perhaps make this comment a non-doc comment:

```
(* The following comment is a documentation comment as it starts with "(**" as opposed to normal comments that start with "(*" *)

(** The first doc comment of this module [Hello] documents the module as a whole, and should contain a description of the module, what it does, how to use it, etc.

  Documentation for individual elements are written next to their declarations. *)